exoctk/exoctk_app/templates/index.html

```diff
@@ -5,7 +5,17 @@
                 <div class="row">
                     <div class="col-md-12">
 
-                        <p style='padding: 0 30px;'>ExoCTK (the Exoplanet Characterization Tool Kit) is an open-source, modular, data analysis software package focused primarily on atmospheric characterization of exoplanets and observation planning. This web application runs the most recent available version of ExoCTK through this web portal designed to single out the most useful tools for observation planning, forward modeling, data reduction, limb darkening, light curve fitting, and retrievals.</p>
+                        <p style='padding: 0 30px;'> The Exoplanet
+                        Characterization ToolKit (ExoCTK) is an open-source,
+                        modular, data analysis software package focused
+                        primarily on the atmospheric characterization of
+                        exoplanets and time-series observation planning. The
+                        web application runs the most recently released version
+                        of ExoCTK through a web portal designed to provide the
+                        most useful tools for observation planning, forward
+                        modeling, data reduction, limb darkening, light curve
+                        fitting, and retrievals. The full version of ExoCTK is
+                        available through GitHub.  </p>
```

Comment:
Looks great, my only comment would be to perhaps add a link to the GitHub repo here?

Reply:
We have it as a tab on the site. Do you think we need it here too?

Reply:
I suppose not!

Reply:
I like the idea of adding a link to the GitHub repo in this text.  Let's do it!